src/System.Private.ServiceModel/tests/Common/Scenarios/Endpoints.cs

```diff
@@ -306,88 +156,79 @@ public static string Https_BasicAuth_Address
     {
         get
         {
-            EnsureLocalClientCertifciateInstalled();
-            return GetEndpointAdddress("BasicAuth.svc//https-basic", protocol:"https");
+            return GetEndpointAddress("BasicAuth.svc//https-basic", protocol:"https");
         } 
     }
 
     public static string Https_DigestAuth_Address
     {
         get
         {
-            EnsureRootCertificateInstalled();
-            return GetEndpointAdddress("HttpsDigest.svc//https-digest", protocol: "https");
+            return GetEndpointAddress("HttpsDigest.svc//https-digest", protocol: "https");
         }
     }
 
     public static string Http_DigestAuth_NoDomain_Address
     {
         get
         {
-            return GetEndpointAdddress("HttpDigestNoDomain.svc//");
+            return GetEndpointAddress("HttpDigestNoDomain.svc//");
         }
     }
 
     public static string Https_NtlmAuth_Address
     {
         get
         {
-            EnsureLocalClientCertifciateInstalled();
```

Comment:
Reviewers: because the [ConditionalFact] detector now calls precisely this method, I believe the right design is to remove these explicit calls from Endpoints.cs.  They would be NOP's and add some confusion if we left them here.